pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go

```diff
@@ -111,6 +111,8 @@ type ProvisionedAlertRule struct {
 	Labels map[string]string `json:"labels,omitempty"`
 	// readonly: true
 	Provenance models.Provenance `json:"provenance,omitempty"`
+	// example: true
```

Comment:
Let's make the example `false` for the docs purposes, as a typical rule is not paused